manim/__main__.py

```diff
@@ -55,26 +54,38 @@ def open_file_if_needed(file_writer):
 
 
 def main():
-    if hasattr(args, "subcommands"):
-        if "cfg" in args.subcommands:
-            if args.cfg_subcommand is not None:
-                subcommand = args.cfg_subcommand
-                if subcommand == "write":
+    args = parse_args(sys.argv)
+
+    if hasattr(args, "cmd"):
+        if args.cmd == "cfg":
+            if args.subcmd:
+                from manim.config import cfg_subcmds
+
+                if args.subcmd == "write":
                     cfg_subcmds.write(args.level, args.open)
-                elif subcommand == "show":
+                elif args.subcmd == "show":
                     cfg_subcmds.show()
-                elif subcommand == "export":
+                elif args.subcmd == "export":
                     cfg_subcmds.export(args.dir)
             else:
-                logger.error("No argument provided; Exiting...")
+                logger.error("No subcommand provided; Exiting...")
+
+        # elif args.cmd == "some_other_cmd":
+        #     something_else_here()
```

Comment:
Did you mean to remove this?

Reply:
Kinda. This is a poor man's documentation saying "this is where you'd put other commands in the future".